templates/web/kingston/waste/biggest_bin_allowed.html

```diff
@@ -0,0 +1,19 @@
+[% USE date(format='%Y%m%d') ~%]
+[% PROCESS 'waste/header.html' %]
+[% IF property %]
+    [% INCLUDE 'waste/_address_display.html' %]
+[% END %]
+
+<h1 class="govuk-heading-xl">Black bin size change request</h1>
+
+[% IF form.saved_data.how_many_exchange == '7more' %]
+<p class="govuk-body">
+If you have an exceptionally large household or your household has medical needs that create more waste than normal, you can apply for more capacity, but this will be assessed by our officers.</p>
+<p class="govuk-body"><a href="https://www.kingston.gov.uk/">Large refuse bin application form</a></p>
```

Comment:
I guess this URL isn't finalised yet :)

Reply:
no i should probably put it in config for ease shouldn't I